frontend/src/HomePage/Header.jsx

```diff
@@ -13,6 +14,29 @@ export default function Header({
   showTemplateLibraryModal,
   fileInput,
 }) {
+
+  const [bgColour, setBgColour] = useState("#ffffff")
+  
+  const dropdownStyle = {
+    minWidth: '11rem',
+    display: 'flex',
+    alignItems: 'center',
+    margin: 0,
+    lineHeight: ' 1.4285714',
+    display: 'block',
+    width: '100%',
+    padding: ' 0.5rem 0.75rem',
+    clear: 'both',
+    fontWeight: '400',
+    color: 'inherit',
+    textAlign: 'inherit',
+    whiteSpace: 'nowrap',
+    background: `${bgColour}`,
+    border: '0',
+    cursor: 'pointer',
+  }
+  const matches = useMediaQuery("(min-width: 1699px)");
```

Comment:
Currently application doesn't support responsiveness, please remove this hook it isn't required at the current stage